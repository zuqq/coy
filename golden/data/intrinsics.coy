fn main() -> () {
    print!("cos(0.0) == {}\n", cos(0.0));
    print!("sin(0.0) == {}\n", sin(0.0));
    print!("sqrt(2.0) == {}\n", sqrt(2.0))
}
