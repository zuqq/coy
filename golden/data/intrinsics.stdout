cos(0.0) == 1.000000
sin(0.0) == 0.000000
sqrt(2.0) == 1.414214
